etc/cas/config/cas.properties

```diff
@@ -90,6 +96,12 @@ cas.logout.remove-descendant-tickets=false
 #
 cas.authn.osf-url.home=https://{{ .Values.osfDomain }}/
 cas.authn.osf-url.dashboard=https://{{ .Values.osfDomain }}/dashboard/
+cas.authn.osf-url.search=https://{{ .Values.osfDomain }}/search/
+cas.authn.osf-url.support=https://help.osf.io
+cas.authn.osf-url.registries=https://{{ .Values.osfDomain }}/registries/discover/
+cas.authn.osf-url.preprints=https://{{ .Values.osfDomain }}/preprints/discover/
+cas.authn.osf-url.meetings=https://{{ .Values.osfDomain }}/meetings/
+cas.authn.osf-url.donate=https://www.cos.io/support-cos
```

Comment:
For cloud request https://centerforopenscience.slack.com/archives/C09AJPR919T/p1758297168789849